src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx

```diff
@@ -43,23 +43,30 @@ const TransactionQueueBar = (): ReactElement | null => {
               unmountOnExit: true,
               mountOnEnter: true,
             }}
+            sx={{
+              // there are very specific rules for the border radius that we have to override
+              borderBottomLeftRadius: '0 !important',
+              borderBottomRightRadius: '0 !important',
```

Comment:
I believe there is a `square` prop on the `Accordion` that can achieve this.

Reply:
But it would unround all the corners, not only the bottom ones, wouldn't it?

Reply:
Valid point.